exercises/go-counting/.meta/src/reference/java/GoCounting.java

```diff
@@ -0,0 +1,156 @@
+import java.awt.Point;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+class GoCounting {
+	int NONE = 0;
+	int WHITE = 1;
+	int BLACK = 2;
+	
+	int[][] board;
```

Comment:
These four variable could (and therefore should since we want to [minimize the accessibility of classes and members](https://github.com/exercism/java/blob/master/POLICIES.md#adhere-to-best-practices)) be private 🙂 

If you want to use the `NONE`, `WHITE` and `BLACK` variables in the tests then please define them as a separate enum, also included as starter implementation (see [the robot-simulatot exercise](https://github.com/exercism/java/blob/master/exercises/robot-simulator/src/main/java/Orientation.java) as an example) 🙂